src/crimson/net/Socket.h

```diff
@@ -39,6 +44,8 @@ class Socket
       // performance. see seastar::net::connected_socket::output()
       out(socket.output(65536)) {}
 
+  ~Socket() { ceph_assert(closed); }
```

Comment:
i think it should be `assert()`.

Reply:
When should use `assert` instead of `ceph_assert`?

Reply:
good q. i think we should use `ceph_assert()` only if we think something *could* happen. `assert()` is for debugging purpose. and i assume that `closed` is added only for debugging.